src/openbiolink/edge.py

```diff
@@ -1,27 +1,41 @@
 from openbiolink.edgeType import EdgeType
+from openbiolink.namespace import Namespace
 
 
 class Edge:
-    def __init__(self, id1: str, type: EdgeType, id2: str, source: "", qScore=None, sourcedb=None):
```

Comment:
why not just include instances of `openbiolink.node.Node`?

Reply:
Did not see that :D thanks!